resources/lang/en/Template.properties

```diff
@@ -377,6 +377,8 @@ notificationsItemBundleSplitted = "Item bundle has been split."
 notificationsItemNotAdded = "The item could not be added."
 notificationsItemOutOfStock = "Insufficient stock"
 notificationsNotEnoughStockItem = "There is not enough stock available for the selected item quantity."
+notificationsBasketItemsRemoved = "Items have been removed from your shopping cart because they are actual not available."
```

Comment:
"Items have been removed from your shopping cart because they are currently not available."